_includes/devdoc/dash-core/rpcs/rpcs/masternode.md

```diff
@@ -607,7 +333,7 @@ The `masternode<!--noref--> status` RPC prints masternode status information.
 
 {% enditemplate %}
 
-*Example from Dash Core 0.12.2*
+*Example from Dash Core 0.13.2*
```

Comment:
reason why not 0.14.0?

Reply:
The change was made in 0.13.2. I just noticed the text hadn't been updated to match. 0.14 didn't modify it any further.